src/screens/AdminPortal/OrgList/modal/OrganizationModal.spec.tsx

```
@@ -131,7 +132,8 @@ describe('OrganizationModal Component', () => {
             formState={formState}
             setFormState={mockSetFormState}
             createOrg={mockCreateOrg}
-          />
+  userData={undefined}
```

ESLint

[error] Insert `··········`

(prettier/prettier)